packages/component/src/Styles/adaptiveCardHostConfig.js

```diff
@@ -1,99 +1,35 @@
 import { HostConfig } from 'adaptivecards';
 import defaultStyleSetOptions from './defaultStyleSetOptions';
-import { primaryFont } from './Fonts';
-
 // https://docs.microsoft.com/en-us/adaptive-cards/rendering-cards/host-config
 
-export default ({ accent } = defaultStyleSetOptions) => new HostConfig({
+export default ({
+  accent,
+  bubbleTextColor ,
+  primaryFont,
+  subtle,
+} = defaultStyleSetOptions) => new HostConfig({
   containerStyles: {
     default: {
       foregroundColors: {
+        default: {
+          default: bubbleTextColor,
+          subtle: subtle
+        },
         accent: {
           default: accent,
-          subtle: accent
+          subtle: subtle
         }
       }
+    },
+    emphasis: {
+      default: {
+        default: bubbleTextColor,
+        subtle: subtle
+      }
     }
   },
```

Comment:
@compulim I'd like your opinion on this.

I made these changes based on the themes in Emulator. I think that allowing users to override the host config handles the extreme case of needing to fully customize the adaptive cards config. This change only passes a few things from the style options through.

Let me know what you think. We could extend this and add more options to styleOptions, but I thought that would not be necessary since the override already exists.